independent-publisher-connectors/FreeAgent/apiProperties.json

```diff
@@ -0,0 +1,82 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Ig_9msmfd2ua93ZVxeO65w",
+          "scopes": [
+
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://api.freeagent.com/v2/approve_app"
+            },
+            "tokenUrl": {
+              "value": "https://api.freeagent.com/v2/token_endpoint"
+            },
+            "refreshUrl": {
+              "value": "https://api.freeagent.com/v2/token_endpoint"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#C4EEAE",

```

Comment:
@nk-gears The default iconBrandColor for all the independent publisher connectors is `"#da3b01"`

Reply:
@sriyen-msft Fixed and Committed.